app/models/parser/twitter_item.rb

```diff
@@ -19,47 +19,63 @@ def patterns
     # Main function for class
     def parse_data_for_parser(_doc, _original_url, _jsonld_array)
       @url.gsub!(/(%23|#)!\//, '')
+      @url.gsub!(/\s/, '')
       @url = replace_subdomain_pattern(url)
+      
       parts = url.match(TWITTER_ITEM_URL)
       user, id = parts['user'], parts['id']
       
-      @parsed_data['raw']['api'] = {}
-      handle_twitter_exceptions do
-        @parsed_data['raw']['api'] = twitter_client.status(id, tweet_mode: 'extended').as_json
+      @parsed_data['raw']['api'] = tweet_lookup(id)
+
+      @parsed_data[:error] = parsed_data['raw']['api']['errors']
+
+      if @parsed_data[:error] 
+        title = url
```

Comment:
**suggestion** I think that we don't need most of this block, since by default the values would be unset and title will be set outside of this class if it's nil. if you want to set the author_url and author_name, maybe we just keep that?